system-test/src/test/resources/TestFile.kt

```diff
@@ -0,0 +1,18 @@
+package org.digma.intellij.plugin.test.system
+
+
+class TestFile {
+
+    fun method1() { }
+
+    fun method2() { }
+    
+    fun relatedSpan1() { }
+    
+    fun relatedSpan2() { }
+    
+    fun relatedEndpoint1() { }
+    
+    fun relatedEndpoint2() { }
+
+}
```

Comment:
does the test also tests spans and endpoints?

Reply:
No, those are for the insights mock data. those methods id's are placed in the Insight as relatedSpan and Endpoint but they real location in the test files for us to navigate to them